### PR TITLE
kubetap submission

### PR DESCRIPTION
**Pull request by Eriner**
Submit Kubetap, https://github.com/soluble-ai/kubetap, to be distributed with krew.

### CODE REVIEW

**Comment by Eriner** [comment]
I signed it

**Comment by Eriner** [comment]
I signed it.

**Comment by Eriner** [comment]
I signed it.

File: plugins/tap.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: tap
+spec:
+  version: v0.1.1
+  homepage: https://soluble-ai.github.io/kubetap/
+  shortDescription: "interactively proxy Kubernetes Services with ease"
```
Comment (ahmetb) [inline plugins/tap.yaml:8]: - please capitalize
- "with ease" is subjective, consider removing

File: plugins/tap.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/soluble-ai/kubetap/releases/download/v0.1.1/kubetap_0.1.1_Darwin_x86_64.tar.gz
+    sha256: 62a7832c7edcff145ec1dc22a04304ac921fc2c9bf723b21142a04d1e6a65a8c
+    bin: kubectl-tap
+    files:
+    - from: "*"
+      to: "."
```
Comment (ahmetb) [inline plugins/tap.yaml:22]: this is the default and not needed, you can remove highlighted portion.

**Comment by ahmetb** [comment]
The CLA still does not seem to be signed. Please try again, if the problem persists, please reach out to the contacts listed in the comment above.

File: plugins/tap.yaml (hunk 2 above)
Comment (Eriner) [reply plugins/tap.yaml:22]: done.

File: plugins/tap.yaml (hunk 1 above)
Comment (Eriner) [reply plugins/tap.yaml:8]: I've added caps. The "with ease" is part of the [project's tagline]( https://github.com/soluble-ai/kubetap) and I have not removed it.

**Comment by Eriner** [comment]
I signed it.

**Comment by Eriner** [comment]
I signed it

**Comment by Eriner** [comment]
I signed it

**Comment by ahmetb** [comment]
/lgtm
/approve
/hold
for another quick review from @corneliusweig

**Comment by corneliusweig** [comment]
Very nice! Thanks for contributing!
/hold cancel

You might want to add a mapping here: https://github.com/kubernetes-sigs/krew/blob/045a61f32a8db19aed8e041ceb750ce89f08929d/cmd/generate-plugin-overview/generate-plugin-overview.go#L121

**Pull request merged**